### ISSUE TITLE
Speech bubble system - Per dialogue/line arguments

### ISSUE THREAD

**Issue by sara5387** [renpy/renpy#4937]
If at all possible, it would be great being able to somehow pass custom bubble parameters via arguments per line of dialogue.
Ex:

```
e "a line of text" (show_trans = bubble_fade)
...
screen bubble(who, what, trans):
    ...
    window:
        if trans is not None:
            at trans 
    ...
```
As of right now, that's only possible by using the **bubble.properties** config variable.

**Comment by renpytom** [comment]
This seems to work fine, at least on the fix branch. (Grab builds from https://nightly.renpy.org/.) Are you not seeing that? 

If so, please post a small game exhibiting the problem. Thanks.

**Comment by sara5387** [comment]
Ah, nice! I tried the latest nightly and it works with inline arguments. Good to know it'll be implemented in 8.2. 😊
